docs/design/capacity-plugin.md

```diff
@@ -0,0 +1,63 @@
+# Capacity Plugin
+
+## Motivation
+
+Capacity plugin is split from proportion, as a long time, capacity of queue works in proportion plugin, proportion plugin is a weight-base plugin, but capacity is not suitable with weight-base.There are contradictions between them.As the total weight increases, the capacity will slowly become invalid.
```

Comment:
Execuse me, may I know the reasons or scenarios why `capacity is not suitable with weight-base`? BTW, I agree with that weight is an attribute of queue. It should be watched and controlled by an independent component such as `queue controller` instead of a plugin. Users can decide to configure the plugin or not. But the queue weight should work all the time without any influence.